packages/data/tests/unit/serializers/metadata/bard-test.js

```diff
@@ -432,14 +479,63 @@ const ParameterConvertToMetricFunction = [
         expression: 'dimension:displayCurrency',
         id: 'currency',
         name: 'currency',
-        type: 'ref',
-        valueType: 'TEXT'
+        type: 'ref'
       }
     ],
     description: '',
     id: 'currency',
     name: '',
     source: 'bardOne'
+  },
+  {
+    id: 'tableName.grain(day,month)',
+    name: 'Time Grain',
+    description: 'Time Grain',
+    source: 'bardOne',
+    _parametersPayload: [
+      {
+        defaultValue: 'day',
+        description: 'The Time Grain to group dates by',
+        expression: 'self',
+        id: 'grain',
+        name: 'Time Grain',
+        source: 'bardOne',
+        type: 'ref',
+        _localValues: [
+          { id: 'day', description: 'The tableName day grain', name: 'Day' },
+          { id: 'month', description: 'The tableName month grain', name: 'MONTH' }
```

Comment:
Is there a reason why we have `Day` and `MONTH` for names? Test normalizing?

Reply:
Just the existing `longName` in the test file, I'm going to leave it for now